addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1390,6 +1391,368 @@ describe('Paste', () => {
+                        await pasteHtml(editor, unformat(`
+                            <ol>
+                                <li class="oe-nested">
+                                    <ol>
+                                        <li class="oe-nested">
+                                            <ol>
+                                                <li class="oe-nested">
+                                                    <ol>
+                                                        <li>abc</li>
+                                                    </ol>
+                                                </li>
+                                                <li>def</li>
+                                            </ol>
+                                        </li>
+                                        <li>ghi</li>
+                                    </ol>
+                                </li>
+                                <li>jkl</li>
+                            </ol>
+                        `));
+                    },
+                    contentAfter: unformat(`  
+                        <ul>
+                            <br>
+                            <li>Alpha</li>
+                            <li>Beta</li>
+                            <li>abc</li>
+                            <li class="oe-nested">
+                                <ul>
+                                    <li class="oe-nested">
+                                        <ul>
+                                            <li>def</li>
+                                        </ul>
+                                    </li>
+                                    <li>ghi</li>
+                                </ul>
+                            </li>
+                            <li>jkl[]</li>
+                        </ul>
+                    `),
+                    //Todo FIXME: Change to
+                    /* contentAfter: unformat(`
+                        ul>
+                            <br>
+                            <li>Alpha</li>
+                            <li>Beta</li>
+                            <li class="oe-nested">
+                                <ul>
+                                    <li class="oe-nested">
+                                        <ul>
+                                            <li class="oe-nested">
+                                                <ul>
+                                                    <li>abc</li>
+                                                </ul>
+                                            </li>
+                                            <li>def</li>
+                                        </ul>
+                                    </li>
+                                    <li>ghi</li>
+                                </ul>
+                            </li>
+                            <li>jkl[]</li>
+                        </ul>
+                    `),
+                    */
```

Comment:
Wait did the spec change and this is what it should do finally? If so that's simple, where you get the deepest `li`, you have to do `querySelector('li')` instead of getting the deepest `li`.

Reply:
It was decided that when we paste a nested list on a empty `li`, it should maintain its nestedness. if we use `querySelector('li')` instead of deepest `li`. This would not work (https://github.com/odoo/odoo/pull/115019/files#r1467491082). We tested it and it was about to fix with task: 3302332

Reply:
Then maybe the commit that fixes that could be integrated here? Since the other task includes more controversial changes that are still being discussed.